tests/rootless.sh

```diff
@@ -100,7 +100,9 @@ function enable_cgroup() {
 		chmod g+rwx "$CGROUP_MOUNT/$CGROUP_PATH"
 		chmod g+rw "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.subtree_control" "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.procs" "$CGROUP_MOUNT/cgroup.procs"
 		# Fix up cgroup.type.
-		echo threaded > "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.type"
+		if grep -qw invalid "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.type"; then
+			echo threaded > "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.type"
+		fi
 		# Make sure cgroup.type doesn't contain "invalid". Otherwise write ops will fail with ENOTSUP.
```

Comment:
You can drop L106-L110 now

Reply:
@lifubang ^^^

Reply:
This has been removed before LGTM.
I think we should keep these comments to let other people know why we need to write threaded to cgroups.type.

Reply:
Is the `echo threaded >` still needed? Seems a bit odd to run the entire test suite under `threaded`.